filebeat/scripts/docs_collector.py

```diff
@@ -1,6 +1,7 @@
 import os
 import argparse
 import yaml
+import six
```

Comment:
Ok, I initially thought based on your comment above that you need six for the encoding. But I read up a bit more on it and it is for the 2x to 3x compatibilitly. Got it.